posts/2023-state-of-rust-analysis.md

```diff
@@ -0,0 +1,71 @@
+
+## Participation
+In 2022, we had 9,433 total survey completions and an increased survey completion rate of 82% vs. 76% in 2021. While the goal is always total survey completion for all participants, the survey requires time, energy, and focus – we consider this figure quite high and were pleased by the increase. 
+
+We also saw a significant increase in the number of people viewing but not participating in the survey (from 16,457 views in 2021 to 25,581 – a view increase of over 55%). While this is likely due to a number of different factors, we feel this information speaks to the rising interest in Rust and the growing general audience following its evolution. 
+
+In 2022, the survey had 11,482 responses, which is a slight decrease of 6.4% from 2021, however, the number of respondents that answered all survey questions has increased year over year. We were interested to see this slight decrease in responses, as this year’s survey was much shorter than in previous years – clearly, survey length is not the only factor driving participation.
+
+## Community
+We were pleased to offer the survey in 11 languages – more than ever before, with the addition of a Ukrainian translation in 2022. 77% of respondents took this year’s survey in English, 5% in Chinese (simplified), 4% in German and French, 2% in Japanese, Spanish, and Russian, and 1% in Chinese (traditional), Korean, Portuguese, and Ukrainian. This is our lowest percentage of respondents taking the survey in English to date, which is an exciting indication of the growing global nature of our community!
+
+The vast majority of our respondents reported being most comfortable communicating on technical topics in English (93%), followed by Chinese (7%). 
+
+Rust user respondents were asked which country they live in. The top 13 countries represented were as follows: United States (25%), Germany (12%), China (7%), United Kingdom (6%), France (5%), Canada (4%), Russia (4%), Japan (3%), Netherlands (3%), Sweden (2%), Australia (2%), Poland (2%), India (2%). Nearly 72.5% of respondents elected to answer this question. 
+
+While we see global access to Rust education as a critical goal for our community, we are proud to say that Rust was used all over the world in 2022!
+
+
+## Rust Usage
+More people are using Rust than ever before! Of our respondents using Rust, 47% do so on a daily basis – an increase of 4% from the previous year. 
+
+30% of Rust user respondents can write simple programs in Rust, 27% can write production-ready code, and 42% consider themselves productive using Rust. 
+
+Over 90% of survey respondents identified as Rust users. Of the former Rust users who completed the survey, 30% cited difficulty as the primary reason for giving up while nearly 47% cited factors outside of their control. 
+
+![1](https://github.com/rust-lang/blog.rust-lang.org/assets/45044840/54c56ffc-2a76-4144-9800-94f5e7da0180)
```

Comment:
I think something might have gone wrong with these images since they are somehow not part of the pull request diff. I assume they should use relative paths to https://github.com/rust-lang/blog.rust-lang.org/tree/master/static/images?

@graciegregory let me know if you need help with that :)